src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Webpack with stylus adds isolation of mixins/functions scope, but not classes. 
We have to manage classes manually since is still global scope, so BEM tries to resolve this issue.